recipes/reduct-cpp/all/patches/1.9.0-0001-use-cci.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index cda9d1d..e6cf1d8 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,9 +14,20 @@ set(REDUCT_CPP_ENABLE_TESTS OFF CACHE BOOL "Compile tests")
+ set(CMAKE_CXX_STANDARD 20)
```

Comment:
This should be commented out for the `compiler.cppstd` setting to take effect :)

Reply:
@RubenRBS 
Thank you for your review!
Fixed.

Reply:
@RubenRBS 
I have a question.
Setting to use C++20 is defined by upstream.
Should we remove such definition from upstream codes on all recipes?